### ISSUE TITLE
Что делать дальше?  

### ISSUE THREAD

**Issue by torvn77** [Kernel-Crowdfunding/New-Linux-Console#4]
Ну вот вы присоединились и молчите, а вместе с тем надо решить, чего будем добиваться, просто исправления ошибок или полного пересмотра кода консоли? 
cast @gremilkar  @bergentroll

**Comment by bergentroll** [comment]
Мой основной интерес — чтобы был исправлен скролл, а вторичный — чтобы у подсистемы был сопровождающий. К сожалению, я некомпетентен, чтобы выдвигать предложения по архитектуре. @torvn77, ваши тех. предложения расцениваю как здравые и полезные.

**Comment by gremilkar** [comment]
Да,присоединяюсь к мнению Bergentroll: "чтобы был исправлен скролл, а вторичный — чтобы у подсистемы был сопровождающий". Предлагаю начать с малого, а потом,как пойдет.
